lib/feature_management.rb

```diff
@@ -103,6 +103,10 @@ def self.log_to_stdout?
     !Rails.env.test? && IdentityConfig.store.log_to_stdout
   end
 
+  def self.prometheus_exporter?
+    !Rails.env.test? && IdentityConfig.store.prometheus_exporter
+  end
+
```

Comment:
instead of adding logic like this, would it be easier to just update the `test:` stanza of the sample application.yml to be false?

Reply:
Well, in the code example (https://github.com/discourse/prometheus_exporter?tab=readme-ov-file#rails-integration), they seem to not want it to be enabled for test stuff, so I was just copying what they were doing.  I also was using the pattern I found in the `log_to_stdout` feature above it to implement that logic.

I'm not sure it's easier or not, but I was trying to do things the way other things are done.  Would you like me to change this?